src/smart-components/user/user.js

```diff
@@ -0,0 +1,229 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import AwesomeDebouncePromise from 'awesome-debounce-promise';
+import { Link } from 'react-router-dom';
+import { Stack, StackItem } from '@patternfly/react-core';
+import { TopToolbar, TopToolbarTitle } from '../../presentational-components/shared/top-toolbar';
+import { Section, DateFormat } from '@redhat-cloud-services/frontend-components';
+import { PrimaryToolbar } from '@redhat-cloud-services/frontend-components/components/PrimaryToolbar';
+import { fetchRoles, fetchRoleForUser } from '../../redux/actions/role-actions';
+import { ListLoader } from '../../presentational-components/shared/loader-placeholders';
+
+import {
+    Table,
+    TableHeader,
+    TableBody,
+    TableVariant,
+    compoundExpand
+} from '@patternfly/react-table';
+
+const columns = [
+    'Roles',
+    {
+        title: 'Groups',
+        cellTransforms: [ compoundExpand ]
+    },
+    {
+        title: 'Permissions',
+        cellTransforms: [ compoundExpand ]
+    },
+    {
+        title: 'Last commit'
+    }
+];
+
+let debouncedFetch;
+
+const User = ({
+    match: { params: { username }},
+    fetchRoles,
+    fetchRoleForUser,
+    roles,
+    isLoading,
+    rolesWithAccess,
+    isRecordLoading
+}) => {
+
+    const [ rows, setRows ] = useState([]);
+    const [ filter, setFilter ] = useState('');
+    const [ page, setPage ] = useState(1);
+    const [ perPage, setPerPage ] = useState(20);
+    const [ expanded, setExpanded ] = useState({});
+
+    const createRows = (data) => {
+        return data ? data.reduce((acc, { uuid, name, groups_in = [], modified, accessCount }, i) => ([
+            ...acc, {
+                uuid,
+                cells: [{ title: name, props: { component: 'th', isOpen: false }},
+                { title: `${groups_in.length}`, props: { isOpen: expanded[uuid] === 1 }},
+                { title: accessCount, props: { isOpen: expanded[uuid] === 2 }},
+                { title: <DateFormat type='exact' date={ modified } /> }
+                ]
+            }, {
+                uuid: `${uuid}-groups`,
+                parent: 3 * i,
+                compoundParent: 1,
+                cells: [{
+                    title: <Table
```

Comment:
You should add props to make the inner table full width `props: { colSpan: 2, className: 'pf-m-no-padding' }`

Reply:
Actually, I've checked and it should have colSpan of parent table, so here it would be 4.